torch/csrc/api/include/torch/detail/TensorDataContainer.h

```diff
@@ -154,7 +148,7 @@ AT_FORALL_SCALAR_TYPES_AND3(Bool, Half, BFloat16, TENSOR)
 
   // NOTE: We need to handle `std::vector` explicitly instead of relying on an implicit conversion
   // to `at::ArrayRef`, otherwise the following error can be thrown when calling
-  // `torch::tensor(std::vector<double>({1.1, 2.2}))`:
+  // `torch::tensor(std::vector<int>({1, 2}))`:
```

Comment:
Why did this change?

Reply:
I made a typo when I wrote this comment originally :/